Assets/MRTK/Examples/Demos/UX/Dwell/BaseDwellPressableButton.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Dwell
+{
+    /// <summary>
+    /// Example script to demonstrate dwell visuals in sample scene
+    /// </summary>
+    public abstract class BaseDwellPressableButton : MonoBehaviour
+    {
+        [SerializeField]
+        protected GameObject dwellVisualImage = null;
```

Comment:
Can we make this of type MeshRenderer, since all instance of this object invoke GetComponent<MeshRenderer>()?

Reply:
That makes sense!

Reply:
Oh, I found that there are some lines that use transform.

Reply:
You can still use the .transform of a MeshRenderer and it will change the transform of the gameobject that owns the component.